README.md

```diff
@@ -1342,6 +1342,8 @@ The user is not blocked and the job is processed in the background.  During this
 
 **[Amazon SQS](https://aws.amazon.com/sqs/)** is hosted but can have high latency and has the possibility of messages being delivered twice.
 
+**[Apache kafka](https://kafka.apache.org/)** is run as a highly available cluster (managed by zookeeper) and is built for realtime data streaming which can be leveraged for event messaging.
```

Comment:
Hi @sudipbhandari126, what's the main downside/trade off?  Note the other alternatives call out trade offs.

Reply:
@donnemartin I have created another PR with the change at https://github.com/donnemartin/system-design-primer/pull/444 (I am closing this.. couldn't update this one because of weird issue I couldn't fix)

Thanks